Compiling error

Having this issue while trying to compile the firmware


```
Compiling: keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c                                  keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c:173:6: error: conflicting types for 'rgb_matrix_indicators_advanced_user'
  173 | bool rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from quantum/quantum.h:36,
                 from keyboards/gmmk/gmmk2/gmmk2.h:19,
                 from keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c:17:
quantum/rgb_matrix/rgb_matrix.h:137:6: note: previous declaration of 'rgb_matrix_indicators_advanced_user' was here
  137 | void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max);
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 [ERRORS]
 |
 |
 |
make[1]: *** [builddefs/common_rules.mk:456: .build/obj_gmmk_gmmk2_p96_ansi_default/keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.o] Error 1
Make finished with errors
make: *** [Makefile:413: gmmk/gmmk2/p96/ansi:default] Error 1
```

This is what I'm getting now

```
Compiling: keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c                                  keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c: In function 'rgb_matrix_indicators_advanced_user':
keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c:217:12: error: 'return' with a value, in function returning void [-Werror=return-type]
  217 |     return false;
      |            ^~~~~
keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.c:173:6: note: declared here
  173 | void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1.exe: all warnings being treated as errors
 [ERRORS]
 |
 |
 |
make[1]: *** [builddefs/common_rules.mk:456: .build/obj_gmmk_gmmk2_p96_ansi_default/keyboards/gmmk/gmmk2/p96/ansi/keymaps/default/keymap.o] Error 1
Make finished with errors
make: *** [Makefile:413: gmmk/gmmk2/p96/ansi:default] Error 1
```

--- a/keymap.c
+++ b/keymap.c
@@ -170,7 +170,7 @@ layer_state_t layer_state_set_user(layer_state_t state) {
 //  119, LED 19
 //  120, LED 20
 
-bool rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
 
     if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK)) {
         RGB_MATRIX_INDICATOR_SET_COLOR(54, 255, 255, 255); //capslock key

--- a/keymap.c
+++ b/keymap.c
@@ -214,7 +214,6 @@ void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
             //rgb_matrix_set_color_all(238, 65, 23);
             //RGB_MATRIX_INDICATOR_SET_COLOR(0, 255, 0, 255);   //Set Color per key
     }
-    return false;
 }
 
 void keyboard_post_init_user(void) {

